src/usr/local/www/services_dhcp_edit.php

```diff
@@ -385,8 +385,7 @@ function staticmaps_sort($ifgui) {
 
 // Get our MAC address
 $ip = $_SERVER['REMOTE_ADDR'];
-$mymac = `/usr/sbin/arp -an | grep '('{$ip}')' | cut -d" " -f4`;
-$mymac = str_replace("\n", "", $mymac);
+$mymac = arp_get_mac_by_ip($ip);
```

Comment:
The new version of arp_get_mac_by_ip() looks good but I was thinking about it and there is still a difference from the current code. arp_get_mac_by_ip() always run a ping to make sure the address will be in the table, and this is not desired in this case because if IP from $_SERVER['REMOTE_ADDR'] is from a local network it will be there.

That ping will introduce 1s delay every time this page is used.

I was thinking it would be nice to add a second parameter to arp_get_mac_by_ip(), default to true, and when it's false don't call ping/ping6.